sched/tls/task_uninitinfo.c

```diff
@@ -50,6 +51,21 @@ void task_uninit_info(FAR struct task_group_s *group)
 {
   FAR struct task_info_s *info = group->tg_info;
 
+#ifdef CONFIG_PTHREAD_ATFORK
+  /* Remove the functions that registered with pthread_atfork() */
+
+  FAR struct list_node *list = &info->ta_atfork;
+  FAR struct pthread_atfork_s *entry;
+
+  while (!list_is_empty(list))
+    {
+      entry = list_first_entry(list,
+                               struct pthread_atfork_s, node);
+      list_delete_init(&entry->node);
+      lib_free(entry);
```

Comment:
the pthread_atfork_s node is allocated by lib_malloc:
```cpp
  FAR struct pthread_atfork_s *entry =
                      (FAR struct pthread_atfork_s *)
                      lib_malloc(sizeof(struct pthread_atfork_s));
```
in case of kernel(not-FLAT build), the memory alloc is called with kmm_malloc, and is correspond with lib_free(kmm_free)

Reply:
But allocation is done from user space while freeing from kernel space, isn't it?

Reply:
> in case of kernel(not-FLAT build), the memory alloc is called with kmm_malloc.


Let me double check, but till now I was expecting that lib_malloc from user space is expanded to malloc and not kmm_malloc